scipy/stats/_continuous_distns.py

```diff
@@ -4518,20 +4518,20 @@ class levy_stable_gen(rv_continuous):
     def _rvs(self, alpha, beta, size=None, random_state=None):
 
         def alpha1func(alpha, beta, TH, aTH, bTH, cosTH, tanTH, W):
-            return (2/np.pi*(np.pi/2 + bTH)*tanTH -
-                    beta*np.log((np.pi/2*W*cosTH)/(np.pi/2 + bTH)))
+            return 2/np.pi*((np.pi/2 + bTH)*tanTH
+                            - beta*np.log((np.pi/2*W*cosTH)/(np.pi/2 + bTH)))
```

Comment:
Yes,
![image](https://user-images.githubusercontent.com/6570539/114280868-2cef1980-99f0-11eb-8025-1e90e66bc747.png)

where
![image](https://user-images.githubusercontent.com/6570539/114280861-25c80b80-99f0-11eb-843d-cc3a458ec1bd.png)

so that factor should be applied to everything. This fixes the problem.